File: app/src/main/java/com/example/aqqhome/Admin_SuachiphiActivity.java
package com.example.aqqhome;

import androidx.appcompat.app.AppCompatActivity;

import android.content.SharedPreferences;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

import com.example.aqqhome.model.feemodel;
import com.example.aqqhome.retrofit.ApiAQQHome;
import com.example.aqqhome.retrofit.RetrofitClient;

import retrofit2.Call;
import retrofit2.Retrofit;

public class Admin_SuachiphiActivity extends AppCompatActivity {
    private ImageView back;

    private Retrofit retrofit;
    private SharedPreferences sharedpreferences;
    private TextView phiquanly, phirac;
    private String ApartmentID;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_admin_suachiphi);
        back = findViewById(R.id.back);
        back.setOnClickListener(v -> {
            finish();
        });
        phiquanly = findViewById(R.id.phiquanly);
        phirac = findViewById(R.id.phirac);
        retrofit = RetrofitClient.getRetrofitInstance();


        sharedpreferences = getSharedPreferences("UserInfo", MODE_PRIVATE);
        String ApartmentID = sharedpreferences.getString("Manager", "");
        ApiAQQHome apiAQQHome = retrofit.create(ApiAQQHome.class);
        Call<feemodel> call = apiAQQHome.admingetchiphi(ApartmentID);
        call.enqueue(new retrofit2.Callback<feemodel>() {
            @Override
            public void onResponse(Call<feemodel> call, retrofit2.Response<feemodel> response) {
                if (response.isSuccessful()) {
                    feemodel text = response.body();
                    if (text != null && text.isSuccess()) {
                        String a = text.getManagementFee();
                        String b = text.getGarbageFee();
                        phiquanly.setText(a);
                        phirac.setText(b);
                    }
                }
            }

            @Override
            public void onFailure(Call<feemodel> call, Throwable t) {

            }
        });



    }
}

File: app/src/main/java/com/example/aqqhome/admin/Admin_SuachiphiActivity.kt
package com.example.aqqhome.admin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.aqqhome.databinding.ActivityAdminSuachiphiBinding
import com.example.aqqhome.model.feemodel
import com.example.aqqhome.retrofit.ApiAQQHome
import com.example.aqqhome.retrofit.RetrofitClient
import retrofit2.Call

class Admin_SuachiphiActivity : AppCompatActivity() {
    private lateinit var binding: ActivityAdminSuachiphiBinding
    private lateinit var apartmentID: String
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityAdminSuachiphiBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.back.setOnClickListener {
            finish()
        }
        val retrofit = RetrofitClient.getRetrofitInstance()
        val sharedpreferences = getSharedPreferences("UserInfo", MODE_PRIVATE)
        apartmentID = sharedpreferences.getString("Manager", "") ?: ""

        val apiAQQHome = retrofit.create(ApiAQQHome::class.java)
        val call = apiAQQHome.admingetchiphi(apartmentID)
        call.enqueue(object : retrofit2.Callback<feemodel> {
            override fun onResponse(call: Call<feemodel>, response: retrofit2.Response<feemodel>) {
                if (response.isSuccessful) {
                    val text = response.body()
                    text?.let {
                        if (it.isSuccess) {
                            binding.phiquanly.text = it.managementFee
                            binding.phirac.text = it.garbageFee
                        }
                    }
                }
            }

            override fun onFailure(call: Call<feemodel>, t: Throwable) {
                // Handle error case
            }
        })
    }
}
